numismatic/cli.py

```diff
@@ -161,64 +161,27 @@ def tabulate(data):
               "to disk")
 # FIXME: The --channel and --api-key-* options are Feed specific and 
 #        should probably be handled differently.
-@click.option('--channel', '-C', default='trades', type=str)
+@click.option('--channels', '-C', multiple=True, default=['trades'])
 @click.option('--api-key-id', default=None)
 @click.option('--api-key-secret', default=None)
 @pass_state
 def listen(state, feed, assets, currencies, raw_output, raw_interval, 
-           channel, api_key_id, api_key_secret):
+           channels, api_key_id, api_key_secret):
     'Listen to live events from a feed'
     feed_name = feed
```

Comment:
Can we rather make the function argument itself `feed_name`. Then this line becomes unnecessary and line 175 looks much nicer (and easier to understand).